starters/pom.xml

```diff
@@ -28,6 +28,7 @@
     <module>starter-kafka-sender</module>
     <module>starter-kafka-receiver</module>
     <module>starter-http-client-rest-async</module>
+    <module>devon4j-starter-http-client-rest-sync</module>
```

Comment:
@SchettlerKoehler noticed that we are breaking our DRY conventions here.
We should therefore remove the `devon4j-` prefix from the folder name as we do for all other starters.